draft-ietf-lake-edhoc.md

```diff
@@ -1383,152 +1384,264 @@ The Initiator and the Responder are allowed to select the connection identifier
 
 
 The sequence of transcript hashes in EDHOC (TH_2, TH_3, TH_4) does not make use of a so called running hash. This is a design choice as running hashes are often not supported on constrained platforms.
 
-When parsing a received EDHOC message, implementations MUST discontinue the protocol if the message does not comply with the CDDL for that message. It is RECOMMENDED to discontinue the protocol if the received EDHOC message is not deterministic CBOR.
+When parsing a received EDHOC message, implementations MUST discontinue the protocol if the message does not comply with the CDDL for that message. Discontinuing the protocol if the received EDHOC message is not deterministic CBOR is RECOMMENDED.
 
 
 # IANA Considerations {#iana}
 
+This Section provides IANA Considerations according to {{RFC8126}}.
+
 ## EDHOC Exporter Label Registry {#exporter-label}
 
-IANA has created a new registry titled "EDHOC Exporter Label" under the new registry group "Ephemeral Diffie-Hellman Over COSE (EDHOC)". The registration procedure is "Expert Review" {{RFC8126}}. The columns of the registry are Label and Description. Label is a uint. Description is a text string. The initial contents of the registry are:
+IANA is requested to create a new registry under the new registry group "Ephemeral Diffie-Hellman Over COSE (EDHOC)" as follows:
 
-<!-- TBD Labels for private use? -->
+Registry Name: EDHOC Exporter Label
 
-~~~~~~~~~~~~~~~~~~~~~~~
-Label: 0
-Description: Derived OSCORE Master Secret
-~~~~~~~~~~~~~~~~~~~~~~~
+Reference: [[this document]]
 
-~~~~~~~~~~~~~~~~~~~~~~~
-Label: 1
-Description: Derived OSCORE Master Salt
-~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~ aasvg
 
++---------+------------------------------+-------------------+
+| Label   | Description                  | Reference         |
++=========+==============================+===================+
+| 0       | Derived OSCORE Master Secret | [[this document]] |
++---------+------------------------------+-------------------+
+| 1       | Derived OSCORE Master Salt   | [[this document]] |
++---------+------------------------------+-------------------+
+| 2-65534 | Unassigned                   |                   |
++---------+------------------------------+-------------------+
+| 65535   | Reserved                     | [[this document]] |
++---------+------------------------------+-------------------+
+| > 65535 | Private Use                  |                   |
++---------+------------------------------+-------------------+
+
+~~~~~~~~~~~
+{: #fig-exporter-label title="EDHOC Exporter Label"}
+
+
+~~~~~~~~~~~
+
++----------------+-------------------------------------+
```

Comment:
Lets have a discussion in LAKE regarding registration procedures.

Reply:
Updated. More reviews are welcome.